internal/command/launch/launch.go

```diff
@@ -309,6 +302,22 @@ func run(ctx context.Context) (err error) {
 		fmt.Fprintf(io.Out, "Created app '%s' in organization '%s'\n", appConfig.AppName, org.Slug)
 	}
 
+	if shouldUseMachines {
+		// Check imported fly.toml is a valid V2 config before creating the app
+		if err := appConfig.SetMachinesPlatform(); err != nil {
+			return fmt.Errorf("Can not use configuration for Apps V2, check fly.toml: %w", err)
+		}
+
+		// remove auto-rollback from machine fly.tomls
+		appConfig.Experimental = nil
+
+		// set default process group
+		// instead of using "", get the entrypoint.sh from the image and stash it there. eg. "nginx -g daemon off" for nginx images
+		if len(appConfig.Processes) == 0 {
+			appConfig.SetProcess(appConfig.DefaultProcessName(), "")
+		}
```

Comment:
this can't be right either 🤔 

the moment we have to force storing a process here means we are not doing the right thing when it is missing on reader side. There are tons of existing fly.tomls without `[processes]` section and this change won't fix the issue for them

Reply:
this felt wrong anyway to be fair lmao